lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -880,6 +880,20 @@ void mme_app_handle_delete_session_rsp(mme_app_desc_t *mme_app_desc_p,
   update_mme_app_stats_s1u_bearer_sub();
   update_mme_app_stats_default_bearer_sub();
 
+  /* If VoLTE is enabled and UE has sent PDN Disconnect
+   * send nas_itti_pdn_disconnect_rsp to NAS.
```

Comment:
Should "nas_itti_pdn_disconnect_rsp" in this comment be "mme_app_itti_pdn_disconnect_rsp"?